### DISCUSSION TITLE
UIUX Polish Review - Mobile Only

### DISCUSSION THREAD

**Discussion by otech47** [setlife-network/trinary#588] [General]
UIUX Polish Review

**Comment by otech47** [comment]
![Screen Shot 2021-12-20 at 11 38 53 AM](https://user-images.githubusercontent.com/4914611/146802813-f00cf0b2-98bd-4895-b477-8b69a1f194fa.png)

**Comment by otech47** [comment]
Reduce margin between payments header and projects list on Client Detail page

**Reply by otech47** [reply]
https://github.com/setlife-network/trinary/issues/591

**Comment by sofiaromorales** [comment]
- [ ] Reduce margin

<img width="200" alt="Screen Shot 2021-12-20 at 12 50 27 PM" src="https://user-images.githubusercontent.com/49292858/146803193-7f36dc91-6fee-490e-a685-c71be37cbfb8.png">

**Comment by otech47** [comment]
On Add Project page

- Align grey numbered circles with the label (Link the Github Project) - remove margin-bottom on numbered circle fixes this
- Add small margin between Project name and Toggl URL in Step 2

**Comment by otech47** [comment]
On Project Detail page

- Reduce margin above and below Overview header text

**Reply by otech47** [reply]
Improvements ProjectOverview summary box

**Comment by otech47** [comment]
On Home Clients List page, Home Projects List page, and Client Detail page (Projects list)

- Client tiles should truncate the client name if it is too long instead of wrapping to a new line
- Project tiles should truncate the project name if it is too long instead of wrapping to a new line

**Comment by sofiaromorales** [comment]
<img width="200" alt="Screen Shot 2021-12-20 at 1 06 27 PM" src="https://user-images.githubusercontent.com/49292858/146805255-57ba45d6-3623-40e3-9253-45dcf5245769.png">

**Comment by otech47** [comment]
Add extra bottom margin to the list of payment allocations on the Project Budgeting page so that the bottom most tiles don't get cut off
![Screen Shot 2021-12-22 at 11 10 44 AM](https://user-images.githubusercontent.com/4914611/147122074-4d50535a-4de0-4e77-beeb-d17af8206404.png)
